### PR TITLE
update feature gate DefaultPodTopologySpread release note

### PR DESCRIPTION
**Pull request by kerthcet**
Signed-off-by: kerthcet <kerthcet@gmail.com>

#### What type of PR is this?
/kind feature

#### What this PR does / why we need it:
We forget to update kube_feature.go in https://github.com/kubernetes/kubernetes/pull/108278

#### Which issue(s) this PR fixes:

Fixes https://github.com/kubernetes/kubernetes/pull/108278

#### Does this PR introduce a user-facing change?

```release-note
None
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs
- [KEP]: https://github.com/kubernetes/enhancements/tree/master/keps/sig-scheduling/1258-default-pod-topology-spread
```

### CODE REVIEW

File: pkg/features/kube_features.go [hunk 1]
```diff
@@ -163,6 +163,7 @@ const (
 
 	// owner: @alculquicondor
 	// beta: v1.20
```
Comment (denkensk) [inline pkg/features/kube_features.go:165]: Alpha is also forgotten.
+ alpha: v1.19

Comment (kerthcet) [reply pkg/features/kube_features.go:166]: Good catch. Thanks Alex.

**Comment by kerthcet** [comment]
/retitle update feature gate DefaultPodTopologySpread release note

**Comment by alculquicondor** [comment]
Thanks!
/lgtm
/approve

**Pull request merged**